### PR TITLE
#165636101 Add Logger to Activities

### PR DESCRIPTION
**Pull request by kevinbett**
## Description (what problem you're fixing)

  - The current implementation does not have loggers to help DevOps engineers troubleshoot the codebase or easily identify points of failure. 

## Fix (what you did to fix it)

  - Implemented loggers to add activity and activity-types endpoints
  - Adds loggers to edit and delete activity types endpoints

## How to test (describe how to test your PR)

- Clone the repository, install docker, run `make test` within the repository.
- Trigger a build on CircleCI.
- Run python manage.py tests

### CODE REVIEW

**Comment by mutoromike** [comment]
Good work @kevinbett. Could you add the logged `activity type` and the `points logged`, it will be easier to find the specific activity that way. In case we need to troubleshoot.

File: src/api/endpoints/activities/activities.py [hunk 1]
```diff
@@ -19,6 +23,7 @@ def __init__(self, **kwargs):
     @roles_required(["success ops", "society president"])
     def post(self):
         """Create an activity."""
+        from manage import app
```
Comment (georgreen) [inline src/api/endpoints/activities/activities.py:26]: is it possible to inject this in the constructor of the view?

File: src/api/endpoints/activity_types/activity_types.py [hunk 2]
```diff
@@ -133,12 +146,14 @@ def delete(self, act_types_id=None):
 
         activity_type = self.ActivityType.query.get(act_types_id)
         if not activity_type:
+            app.logger.warning('Activity not found!')
```
Comment (georgreen) [inline src/api/endpoints/activity_types/activity_types.py]: It might be of useful to log the Id of the activity that was not found.

**Review by georgreen — COMMENTED** [review]
Looks good, you could add the objects info and the user id of the user who attempted the operation.

**Comment by kevinbett** [comment]
@georgreen I have updated the logger with the objects info and user info when an operation is attempted.

**Review by mutoromike — APPROVED** [review]
LGTM.

**Review by frankip — APPROVED** [review]
LGTM

File: src/api/endpoints/activities/activities.py (hunk 1 above)
Comment (kevinbett) [reply src/api/endpoints/activities/activities.py:26]: Hi @georgreen I tried implementing this approach but it fails to execute when injected into the constructor class. Can I go ahead and merge the PR as is?

Comment (georgreen) [reply src/api/endpoints/activities/activities.py:26]: @kevinbett Sure, just add a TODO comment explaining `why we have that and we should try to remove it there in future refactors.` Then you can proceed to merge.

Comment (kevinbett) [reply src/api/endpoints/activities/activities.py:26]: Okay sure.

**Pull request merged**